posthog/models/session_recording_event/sql.py

```diff
@@ -18,6 +18,7 @@
     session_id VARCHAR,
     window_id VARCHAR,
     snapshot_data VARCHAR,
+    events_summary VARCHAR,
```

Comment:
I'm wondering if it makes more sense to use and `Array(VARCHAR)` or `JSON()` field here? I haven't played much with these, but I imagine if we want to use this field in the list query, it'd be faster if it's structured.